src/main/java/dook/command/AddTaskCommand.java

```diff
@@ -0,0 +1,31 @@
+package dook.command;
+
+import dook.DookException;
+import dook.services.Storage;
+import dook.services.TaskList;
+import dook.services.UiDisplay;
+import dook.task.Task;
+
+/**
+ * Command for adding a task to the task list.
+ */
+public class AddTaskCommand extends Command {
+    private final Task task;
+    public AddTaskCommand(Task task) {
```

Comment:
perhaps can add a newline between the field and the method for better readability? I noticed the same issue in several other places too

Reply:
Definitely agree, thanks for the suggestion!